prh/ja.yml

```
@@ -31,3 +31,6 @@ rules:
   - expected: プログラム
     patterns:
       - プログラマティック
+  - expected: 伝搬
+    patterns:
+      - 伝播
```

[imo]

@kohbis cc @open-telemetry/docs-ja-approvers 

プロパゲーションも追加して良さそうです。
content/ja/docs/concepts/context-propagation.md の`伝搬（プロパゲーション）`は`伝搬`にしちゃって良いですかね。

```suggestion
      - 伝播
      - プロパゲーション
```